libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/guidance/signboard/api/MapboxSignboardApiTest.kt

```diff
@@ -234,6 +235,7 @@ class MapboxSignboardApiTest {
         assertEquals(expected.value.bytes, messageSlot.captured.value.bytes)
     }
 
+    @Ignore("Make this test an instrumentation test to avoid UnsatisfiedLinkError from Common 11+")
```

Comment:
For now, ignoring this test to avoid `UnsatisfiedLinkError`s brought by Common `11+`.

cc @LukasPaczos @kiryldz @pengdev @abhishek1508